cppsdk/include/playfab/PlayFabEventApi.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#ifndef DISABLE_PLAYFABENTITY_API
+
+#include <playfab/PlayFabEvent.h>
+#include <playfab/PlayFabEventRouter.h>
+
+namespace PlayFab
+{
+    /// <summary>
+    /// Main interface for PlayFab Sdk, specifically all Lightweight/Heavyweight Event APIs
+    /// </summary>
+    class PlayFabEventAPI
+    {
+    public:
+        PlayFabEventAPI();
+        void EmitEvent(std::unique_ptr<const IPlayFabEvent> event, const PlayFabEmitEventCallback callback) const;
```

Comment:
Yes, since we are moving the argument of the function and not changing any of the members of our object itself. All this code in PR compiles.

---
In reply to: [227947711](https://github.com/PlayFab/XPlatCppSdk/pull/6#discussion_r227947711) [](ancestors = 227947711)